### DISCUSSION TITLE
Copy Elements from any website as Tailwind CSS components

### DISCUSSION THREAD

**Discussion by RealBrianH** [tailwindlabs/tailwindcss#11261] [Show and tell]
Hello!

Would you like to convert any website on the internet into Tailwind CSS?

I wanted to share with you a tool that I created ([DivMagic](https://divmagic.com)) which lets you copy any element from any website as a Tailwind CSS component.

It includes support for most of Tailwind's common features. It will convert elements directly into Tailwind CSS (including colors!)

![demo-1](https://github.com/tailwindlabs/tailwindcss/assets/119697666/0673f3fd-61b3-40e5-88b0-ce558c748902)

Check it out at [divmagic.com](https://divmagic.com) and let me know what you think!

If you find any issues, you can raise them [here](https://github.com/DivMagicCom/DivMagic/issues).

**Comment by yanggggjie** [comment]
Hi, you made a super cool extension! I am new in FE development, it will help me to practice writing pages.
